src/overlays/actors/ovl_Train_Window/ac_train_window.c

```diff
@@ -300,7 +300,6 @@ void Train_Window_Actor_draw(Actor* thisx, Game_Play* play) {
     Train_Window* this = (Train_Window*)thisx;
     UNUSED s32 pad;
 
```

Comment:
With these macros you should be able to 
1. Remove the `if(1){}`s from `Train_Window_Actor_draw`
2. Remove the `if(1)` block in `aTrainWindow_DrawGoingOutTunnel`

and if you remove the else (since the if ends in a `return 0`), you can also remove the `if (!this->paletteId) {}`. Same trick seems to work in all functions with that pattern.

Reply:
Thanks! That seems to have done the trick.